sphinx/writers/latex.py

```diff
@@ -1507,10 +1524,18 @@ def get_nested_level(node):
             else:
                 return get_nested_level(node.parent)
 
+        enum = "enum%s" % toRoman(get_nested_level(node)).lower()
+        enumnext = "enum%s" % toRoman(get_nested_level(node) + 1).lower()
+        style = ENUMERATE_LIST_STYLE.get(get_enumtype(node))
+
         self.body.append('\\begin{enumerate}\n')
+        self.body.append('\\def\\the%s{%s{%s}}\n' % (enum, style, enum))
+        self.body.append('\\def\\label%s{%s\\the%s %s}\n' % 
```

Comment:
I use ``\\the%s %s`` and not ``\the%s%s`` in case suffix is a letter. I know this is perhaps not possible. Anyhow, for LaTeX the space after the control sequence is gobbled.

Reply:
+1